scripts/tests/test_validate_docstrings.py

```diff
@@ -50,14 +49,14 @@ def unused_import(self) -> None:
         Examples
         --------
         >>> import pandas as pdf
-        >>> df = pd.DataFrame(np.ones((3, 3)), columns=('a', 'b', 'c'))
+        >>> df = pd.DataFrame(np.ones((3, 3)), columns=("a", "b", "c"))
         """
 
     def missing_whitespace_around_arithmetic_operator(self) -> None:
         """
         Examples
         --------
-        >>> 2+5
+        >>> 2 + 5
```

Comment:
You are fixing those, which are incorrect docstrings used to test incorrect docstrings. All these changes need to be reverted. This is why the CI is red for your PR.